vmf/scripts/mods/vmf/modules/core/network.lua

```diff
@@ -385,9 +388,21 @@ vmf.create_network_dictionary = function()
   _shared_mods_map = cjson.encode(_shared_mods_map)
   _shared_rpcs_map = cjson.encode(_shared_rpcs_map)
 
+  if not VT1 then
+    Managers.mod:network_bind(0, function(sender, payload)
```

Comment:
Should this 0 be VT2_PORT_NUMBER?
```suggestion
    Managers.mod:network_bind(VT2_PORT_NUMBER, function(sender, payload)
```